geode-protobuf/src/main/java/org/apache/geode/protocol/protobuf/operations/GetAllRequestOperationHandler.java

```diff
@@ -59,13 +59,14 @@
       }
       return Success.of(RegionAPI.GetAllResponse.newBuilder().addAllEntries(entries).build());
     } catch (UnsupportedEncodingTypeException ex) {
-      return Failure.of(BasicTypes.ErrorResponse.newBuilder()
-          .setErrorCode(ProtocolErrorCode.VALUE_ENCODING_ERROR.codeValue)
-          .setMessage("Encoding not supported.").build());
+      int errorCode = ProtocolErrorCode.VALUE_ENCODING_ERROR.codeValue;
+      String message = "Encoding not supported.";
+      return Failure.of(ProtobufResponseUtilities.makeErrorResponse(errorCode, message));
```

Comment:
good catch! The variables were supposed to be temporary to enable IntelliJ's refactoring.